src/chunk.c

```diff
@@ -1266,6 +1266,7 @@ chunk_get_chunks_in_time_range(Oid table_relid, Datum older_than_datum, Datum ne
 
 		time_dim = hyperspace_get_open_dimension(ht->space, 0);
 
+		// FIXME should this be partitioning_type instead?
```

Comment:
this should probably be fixed or removed :)

Reply:
@svenklemm @cevian  well should it? (if we don't know, the `FIXME` is better than having us forget about it)

Reply:
oh i thought you left this as todo for yourself

Reply:
i think it should be partitioning_type though because column_type makes no sense

Reply:
@svenklemm mayyybe, but that would be a strictly weaker condition than we have here; it would allow multiple chunks which have different time-column types to be dropped together.

Honestly, I'm inclined to say that allowing `drop_chunks` to affect multiple tables is a miss-feature, and I don't care if it works; pretty much anything will break it anyway.